package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@codahq/packs-sdk",
-  "version": "1.7.11",
+  "version": "1.7.12",
```

Comment:
since you'll be releasing this immediately, this is fine, but for future reference, the ideal flow is to make this `1.7.12-prerelease.1` and then as a part of doing the release process, it gets to drop the prerelease.

Reply:
I'll update this